AppVuelos/src/screens/test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View, Text, Pressable} from 'react-native';
+import auth, {FirebaseAuthTypes} from '@react-native-firebase/auth';
+import { useNavigation } from '@react-navigation/native';
+import handleLogOut from '../hooks/HandleLogOut';
+
+function Test(): JSX.Element {
+  const navigation = useNavigation<any>();
```

Comment:
Remember to rename this component as the final version ^^

Reply:
sure sure, was only a test.